src/menus/FileMenus.cpp

```diff
@@ -34,6 +34,35 @@
 // private helper classes and functions
 namespace {
 
+bool ExportWithPrompt(AudacityProject &project,
+                      Exporter &exporter,
+                      const FileExtension& format,
+                      double t0, double t1, bool selectedOnly)
+{
+   bool skipSilenceAtBeginning;
+   gPrefs->Read(wxT("/AudioFiles/SkipSilenceAtBeginning"),
+                                   &skipSilenceAtBeginning, false);
+   
+   if(!exporter.SetExportRange(t0, t1, false, skipSilenceAtBeginning))
```

Comment:
Did you mean to pass `selectedOnly` for the third argument?

Reply:
Yes, thanks!